src/routes/reset-password/components/ResetPasswordPage.js

```diff
@@ -0,0 +1,85 @@
+const { useState } = require('react');
+const T = require('prop-types');
+const { default: Styled } = require('styled-components');
+const { default: Typography } = require('@mui/material/Typography');
+const { default: TextField } = require('@mui/material/TextField');
+const { default: Button } = require('@mui/material/Button');
+const { default: Box } = require('@mui/material/Box');
+
+const internals = {};
+
+module.exports = function ResetPasswordPage({ onPressResetPassword, isAuthenticated }) {
+
+    const [password, setPassword] = useState('');
+    const [isSubmitting, setIsSubmitting] = useState(false);
+
+    const disableSubmit = () => {
+
+        return !password;
+    };
+
+    const handleSubmit = async (ev) => {
+
+        ev.preventDefault();
+        const accountInfo = formatFields();
+        setIsSubmitting(true);
+        const [err] = await onPressResetPassword(accountInfo);
+        setIsSubmitting(false);
+        if (!err) {
+            window.location.href = '/login';
```

Comment:
Could we use the middle end here, too? Maybe by dispatching a `push` action?

Reply:
➕ 1️⃣